src/cadenza.js

```diff
@@ -555,6 +555,8 @@ export class CadenzaClient {
    * @param {boolean} [options.useMapSrs] - Whether the created geometry should use the map's SRS (otherwise EPSG:4326 will be used)
    * @param {OperationMode} [options.operationMode] - The mode in which a workbook should be operated
    * @param {AbortSignal} [options.signal] - A signal to abort the iframe loading
+   * @param {Object} [options.additionalLayerDefinitions] - Layer definitions to be imported and shown in the background, as a basis for the drawing.
+   * @param {Object[]} [options.additionalLayerDefinitions.additionalLayers] - Layer definition, with name, type and content (a Geojson featureCollection).
```

Comment:
For the param name we thought of these possibilities:
- additionalLayers
- referenceLayers
- adhocLayers

Reply:
Why is there an intermediate "additionalLayerDefinitions" element?

Reply:
It seemed like we needed a json but I see now we don't so I'll remove it.